src/Pages/Settings/Addons.elm

```diff
@@ -65,12 +68,116 @@ view : Model -> Document Msg
 view model =
     { title = "Settings.Addons"
```

Comment:
This title is not how we want it to be in the end, so you could fix that (and similar ones)

Reply:
ok got it.